api/models/enrollment_tests.py

```diff
@@ -86,20 +79,13 @@ def __str__(self) -> str:
         return f"Test results of {self.student}"
 
     @property
-    def right_answers_percentage(self) -> float:
-        """
-        Get percentage of correct answers.
-        """
+    def passed_level(self) -> str:
+        """Returns language level of the student based on amount of right answers."""
+        right_answers = self.answers.filter(is_correct=True).count()
```

Comment:
maybe `correct_answers` was good naming after all, since it matches `is_correct`.

Reply:
fixed